setup.yml

```diff
@@ -13,6 +13,7 @@
     - flannel
     - master
     - addons
+    - dnsmasq
```

Comment:
Sequence if important, because until we don’t have kube-dns, dnsmasq setup is partially functional. If it’s acceptable we can make dnsmasq dependency of kubernetes or even addons.

> On 21 Aug 2015, at 14:47, Kenny Jones <notifications@github.com> wrote:
> 
> In setup.yml <https://github.com/CiscoCloud/kubernetes-ansible/pull/54#discussion_r37627057>:
> 
> > @@ -13,6 +13,7 @@
> >      - flannel
> >      - master
> >      - addons
> > +    - dnsmasq
> does the sequencing of when to install and configure dnsmasq need to be last or maybe just a simple dependency to the role kubernetes since both master and minion require it?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/CiscoCloud/kubernetes-ansible/pull/54/files#r37627057>.
>